src/components/home/DialogSelect.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '@material-ui/core/Button';
+import { withStyles } from '@material-ui/core/styles';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import InputLabel from '@material-ui/core/InputLabel';
+import Input from '@material-ui/core/Input';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select'; import { FileCopy } from '@material-ui/icons';
+
+const styles = theme => ({
+    container: {
+        display: 'flex',
+        flexWrap: 'wrap',
+    },
+    formControl: {
+        margin: theme.spacing.unit,
+        minWidth: 120,
+    },
+});
+
+class DialogSelect extends React.Component {
+    state = {
+        open: false,
+        age: '',
+        back2Back: null,
+        noShow: null
```

Comment:
<img width="206" alt="screen shot 2018-12-26 at 12 44 18 pm" src="https://user-images.githubusercontent.com/16419909/50453081-282ff580-090c-11e9-869e-be98934e856a.png">

Setting the values to `null` results in a lot console warnings. 

Warning Example: 
<img width="727" alt="screen shot 2018-12-26 at 12 46 08 pm" src="https://user-images.githubusercontent.com/16419909/50453112-4f86c280-090c-11e9-8f7b-8335a6d9e389.png">


Suggestion:
Let's set them to empty string
<img width="171" alt="screen shot 2018-12-26 at 12 47 38 pm" src="https://user-images.githubusercontent.com/16419909/50453147-878e0580-090c-11e9-95b9-dd98749d48b4.png">